### ISSUE TITLE
[Brokernode] Attachment seems to hang when using lambdas in the prod environment

### ISSUE THREAD

**Issue by rfornea** [oysterprotocol/brokernode#576]
The logic for how the tasks run has been switched around.  Previously, verification could happen at the same time as attachment.  I.e. the broker could perform verification on a file while attaching for that file is still in progress.  The logic is no longer like that, but back when it was, the broker would keep resetting the NextIdxToAttach back to previous values because it did not see the transactions on the tangle.  This would cause it to keep re-PoWing the same chunks over an over, when lambda was enabled.

Currently the logic goes through and attaches everything before it bothers verifying anything.  But the polling on the front end for large uploads stops detecting new transactions partway through, with lambda enabled.  

Basically it seems that either the transactions don't successfully get attached after a certain point with lambda enabled.  Or it could be that the lambdas stop completing.
